backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/commons/tickets.js

```diff
@@ -66,6 +66,26 @@ const templateIDToParams = async (req, res, next) => {
 	}
 };
 
+TicketsMiddleware.convertFilterToProjection = async (req, res, next) => {
```

Comment:
as discussed, move this back to processor, and call split on the filter before calling the processor function